index.html

```diff
@@ -0,0 +1,330 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="description" content="This is portfolio repository" />
+    <meta
+      name="author"
+      content="Bhaskar Ranjan Bora, B.Tech Computer Science"
+    />
+    <title>Bhaskar Ranjan Bora</title>
+
+    <link href="css/main.css" rel="stylesheet" />
+
+    <link
+      href="https://fonts.googleapis.com/css?family=Poppins"
+      rel="stylesheet"
+    />
+
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+  </head>
+  <body>
+    <main>
+      <header>
+        <nav>
+          <div class="logo">
+            <h1>My Logo</h1>
+          </div>
+          <div class="button">
+            <i class="fa fa-bars fa-2x"></i>
+          </div>
+          <div class="destop-menu">
+            <ul>
+              <li>Portoflio</li>
+              <li>About</li>
+              <li>Contact</li>
+            </ul>
+          </div>
+        </nav>
+      </header>
+      <section class="section1">
+        <div class="hero-section">
+          <div class="child">
+            <div class="title">
+              <h1>I'm Bhaskar,<br />Glad to see you!</h1>
+            </div>
+            <div class="description">
+              <p>
+                I’m a software developer! I can help you build a product,
+                feature or website Look through some of my work and experience!
+                If you like what you see and have a project you need coded,
+                don’t hestiate to contact me.
+              </p>
+            </div>
+
+            <div class="social">LET’S CONNECT</div>
```

Comment:
- [ ] kindly update the `LET’S CONNECT` color to #7F8CFF so you match the Figma design and since its a title kindly wrap it inside a heading tag `h2 ... h6` to be semantically correct